src/particles/PhysicsParticles.hpp

```diff
@@ -185,11 +189,16 @@ class PhysicsParticleDescriptorBase
 	[[nodiscard]] AMREX_GPU_HOST_DEVICE auto getMassIndex() const -> int { return massIndex_; }
 	[[nodiscard]] AMREX_GPU_HOST_DEVICE auto getLumIndex() const -> int { return lumIndex_; }
 	[[nodiscard]] AMREX_GPU_HOST_DEVICE auto getBirthTimeIndex() const -> int { return birthTimeIndex_; }
+	[[nodiscard]] AMREX_GPU_HOST_DEVICE auto getParam1() const -> double { return param1_; }
+	[[nodiscard]] AMREX_GPU_HOST_DEVICE auto getParam2() const -> double { return param2_; }
```

Comment:
Same issue as before -- does this class need to be modified just for the test problem?